internals/secrethub/flags.go

```diff
@@ -15,5 +15,5 @@ func registerTimestampFlag(r FlagRegisterer) *kingpin.FlagClause {
 }
 
 func registerForceFlag(r FlagRegisterer) *kingpin.FlagClause {
-	return r.Flag("force", "Ignore confirmation and fail instead of prompt for missing arguments.").Short('f')
+	return r.Flag("force", "Disable all confirmation prompts.").Short('f')
```

Comment:
Please include that this overwrites the output file if it already exists. See this ticket: https://github.com/secrethub/secrethub-cli/issues/261

Reply:
Isn't that specific to the `secrethub inject` command?

Reply:
Yes correct. Should have been more explicit about that. My suggestion is to not use this function, but instead register the same flag with a more explicit message that explains exactly what it means for the `inject` command.

Reply:
Got it, thanks!

Reply:
Please also include Marc's suggestion of making explicit that the `--force` flag is ignored if no `--output-file` is supplied. See the ticket: https://github.com/secrethub/secrethub-cli/issues/261#issuecomment-601125693

Reply:
Done